example/warpctc/lstm.py

```diff
@@ -0,0 +1,134 @@
+# pylint:skip-file
```

Comment:
please add a readme to this example and mention that it need the warpctc plugin to work.